packages/base/Select/src/NonNativeSelect/NonNativeSelect.tsx

```diff
@@ -32,12 +30,19 @@ import {
   filterFlatOptions as defaultFilterOptions,
 } from '../SelectBase'
 import { NonNativeSelectOptions } from '../NonNativeSelectOptions'
-import styles from './styles'
 import { NonNativeSelectLimitFooter } from '../NonNativeSelectLimitFooter'
-const useStyles = makeStyles<Theme>(styles)
 
 const DEFAULT_EMPTY_ARRAY_VALUE: ValueType[] = []
 
+const rootWidthClassNames: Record<
```

Comment:
```suggestion
const classesByWidth: Record<
```
NIT: just for consistency with other components